test/gui/shared/steps/steps.py

```diff
@@ -724,3 +735,19 @@ def step(context, resource, group):
 
     sharingDialog = SharingDialog()
     sharingDialog.selectCollaborator(group, True)
+
+
+@When('the user overwrites the file "|any|" with content "|any|"')
+def step(context, resource, content):
+    print("before overwrite")
```

Comment:
remove the print

Reply:
it is for the log purpose that would be visible in CI as well

Reply:
ok, then can you write a different message something like `starting file overwrite` and `file has been successfully overwritten`?